install.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/sh
+
+TID_ARCH=""
+TID_OS=""
+
+TID_DIR="$HOME/.tid"
+TID_BIN_DIR="$TID_DIR/bin"
+TID_VERSION=""
+
+prepareArch() {
+	TID_ARCH=$(uname -m)
+
+	case "$TID_ARCH" in
+		x86) TID_ARCH="386";;
+		x86_64) TID_ARCH="amd64";;
+		i686) TID_ARCH="386";;
+		i386) TID_ARCH="386";;
+	esac
+}
+
+prepareOS() {
+	TID_OS=$(echo `uname`|tr '[:upper:]' '[:lower:]')
+
+	case "$TID_OS" in
+		# Minimalist GNU for Windows
+		mingw*) TID_OS='windows';;
+	esac
+}
+
+prepareTidDir() {
+    if test ! -d "$TID_DIR"; then
```

Comment:
i'm not sure if it's something to do with github or not, but the indentation of these ifs in these methods looks off?